R/utils.R

```diff
@@ -48,24 +48,13 @@ two_cols_match <- function(col1, col2, data) {
   all(unlist(data[col1]) == unlist(data[col2]))
 }
 
-odd <- function(x) {
```

Comment:
are odd_index/even_index still used?

Reply:
Yes, they are.

Reply:
btw I wonder if {lintr} should offer such a check, i.e. if all symbols are used. Does it already?